CONTRIBUTING.md

```diff
@@ -48,6 +48,18 @@ the platform, and provide a unique non-trivial challenge to modern
 machine learning algorithms. Feel free to submit these environments with a
 PR explaining the nature of the environment and task.
 
-## Style Guide
+## Continuous Integration (CI)
```

Comment:
I think this is still a style guide, we just enforce our style guide through CI.

Reply:
There's a "Code Style" section below. I wanted to give a little more explanation about how the checks (black, mypy, and anything else we add later) were run. I can change this back and move it to another doc if you think it was better before.

Reply:
No I think everything is fine here. I don't have a strong opinion on this.